query-dsl/src/main/java/org/infinispan/query/dsl/SearchTimeoutException.java

```diff
@@ -0,0 +1,16 @@
+package org.infinispan.query.dsl;
+
+/**
+ * Thrown when a query timeout occurs.
+ *
+ * @since 11.0
+ */
+public class SearchTimeoutException extends RuntimeException {
```

Comment:
I guess the days of dsl package are numbered. could we maybe move this to top level query package ?

Reply:
moved

Reply:
Ok. Now I'm going to worry about split packages :)